rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -26,6 +26,34 @@
 #pragma GCC diagnostic ignored "-Wuninitialized"
 #endif
 
+#ifdef RPCS3_HAS_MEMORY_BREAKPOINTS
+void ppubreak(ppu_thread& ppu)
+{
+	if (!g_breakpoint_handler.IsBreakOnBPM()) return;
+
+	bool status = ppu.state.test_and_set(cpu_flag::dbg_pause);
+#ifdef WITH_GDB_DEBUGGER
+	fxm::get<GDBDebugServer>()->notify();
```

Comment:
This won't compile, we use FXO now.

Reply:
I removed that whole ifdef, I couldn't find traces of GDBDebugServer or WITH_GDB_DEBUGGER in the rest of the source code.